### DISCUSSION TITLE
Ligatures?

### DISCUSSION THREAD

**Discussion by smolck** [Chlumsky/msdf-atlas-gen#29] [Q&A] [answered]
For fonts that support it, is there any way to, say, pass `->` or `=>` or similar as a single unit to the font atlas to enable drawing a ligature? Or how can those be supported when rendering with MSDFs, if such support is possible?

Apologies if this is out of place on this repo, but I'm really lost as to how to support this sort of thing. I've looked all over the place, and can't seem to find any information on the topic that's comprehensive, or having a clear application to this library (like harfbuzz).

[This issue](https://github.com/Chlumsky/msdfgen/issues/105#issue-618164191) in msdfgen briefly mentions ligatures, but again it remains unclear to me after much searching how to use this for ligature rendering.

**Comment by Chlumsky** [comment, marked as answer]
To use ligatures and other glyphs that don't directly map to Unicode characters, you need to use a *glyph set* instead of a charset. That is, either using the `-glyphset` argument, or the `FontGeometry::loadGlyphset` function. You would use a library such as Harfbuzz to get the appropriate glyph indices.

**Reply by smolck** [reply]
That works, thank you!
